.github/workflows/intel.yml

```diff
@@ -33,20 +33,20 @@ jobs:
     - name: configure
       run: |
         source /etc/profile
-        module load intel-oneapi-compilers intel-oneapi-dpl intel-oneapi-mkl intel-oneapi-tbb cmake
+        module load intel-oneapi-compilers intel-oneapi-dpl intel-oneapi-mkl cmake
         mkdir build
         cd build
         cmake .. -DCMAKE_INSTALL_PREFIX=install_ginkgo -DCMAKE_CXX_FLAGS="-Wpedantic -ffp-model=precise" -DCMAKE_CXX_COMPILER=${{ matrix.config.compiler }} -DCMAKE_BUILD_TYPE=${{ matrix.config.build_type }} -DGINKGO_MIXED_PRECISION=${{ matrix.config.mixed }} -DGINKGO_BUILD_CUDA=OFF -DGINKGO_BUILD_HIP=OFF -DGINKGO_BUILD_MPI=OFF -DGINKGO_DPCPP_SINGLE_MODE=ON
         make -j8
         ONEAPI_DEVICE_SELECTOR=level_zero:gpu ctest -j10 --output-on-failure
-
+ 
     - name: install
       run: |
         source /etc/profile
-        module load intel-oneapi-compilers intel-oneapi-dpl intel-oneapi-mkl intel-oneapi-tbb cmake
+        module load intel-oneapi-compilers intel-oneapi-dpl intel-oneapi-mkl cmake
         cd build
         SYCL_DEVICE_FILTER=level_zero:gpu make install
         export GINKGO_PATH="$(pwd)/install_ginkgo/lib"
-        export LIBRARY_PATH=${GINKGO_PATH}:$LIBRARY_PATH
-        export LD_LIBRARY_PATH=${GINKGO_PATH}:$LD_LIBRARY_PATH
+        export LIBRARY_PATH=${ICL_INTEL_TBB_ROOT}/lib64:${GINKGO_PATH}:$LIBRARY_PATH
+        export LD_LIBRARY_PATH=${ICL_INTEL_TBB_ROOT}/lib64:${GINKGO_PATH}:$LD_LIBRARY_PATH
```

Comment:
This seems a bit brittle, but as long as it works it's fine with me.
Have you tried using `find_package(...)` and `link_libraries(...)` to set the paths instead?

Reply:
yes, I agree with you.
No, I haven't. I think it should be done from MKL/DPL if they put it in public linking interface.
There are two package name (for the same thing, AFAIK): TBB from oneMKL (current in machine) and oneTBB from usual setup.